### PR TITLE
docs: refactor/doc slots

### LINKED ISSUE #6440: [Bug Report] v-tooltip does not work when activator is set using the new slot syntax
### Versions and Environment
**Vuetify:** 1.5.0
**Last working version:** 1.4.0
**Vue:** 2.6.4
**Browsers:** Firefox 67.0, Microsoft Edge, Mozilla Firefox, Google Chrome
**OS:** Windows 10, Linux

### Steps to reproduce
Change the template from `<template slot="activator">` to `<template #activator>`.

### Expected Behavior
Tooltip still appears when activator is hovered.

### Actual Behavior
Tooltip does not appear.

### Reproduction Link
<a href="https://jsfiddle.net/d9qc0n26/1/" target="_blank">https://jsfiddle.net/d9qc0n26/1/</a>

### PR DESCRIPTION
**Pull request by MajesticPotatoe**
## Description
Update docs and doc examples to use new slot syntax

## Motivation and Context
Because slot life is best life
Closes #6440

## How Has This Been Tested?
Visually, Lint

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:
- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by DRoet** [comment]
The usage of `v-slot` and the shorthand syntax `#` seems inconsistent, no ?

**Comment by MajesticPotatoe** [comment]
> The usage of `v-slot` and the shorthand syntax `#` seems inconsistent, no ?

The # syntax is only needed for dialog/menu/tooltip activators. everything else uses v-slot.

**Comment by KaelWD** [comment]
They're the same thing, like `v-on` and `@`

File: packages/docs/src/components/core/Ecosystems.vue [hunk 1]
```diff
@@ -6,19 +6,21 @@
...
-      />
-      <v-icon class="hidden-sm-and-down">mdi-menu-down</v-icon>
-      <v-icon class="hidden-md-and-up">mdi-earth</v-icon>
-    </v-btn>
+    <template #activator="{ on: menu }">
+      <v-btn
+        :aria-label="$t('Vuetify.AppToolbar.ecosystem')"
+        flat
+        style="min-width: 48px"
+        v-on="{ ...menu }"
```
Comment (KaelWD) [inline packages/docs/src/components/core/Ecosystems.vue]: don't need to spread if there's only one

**Comment by MajesticPotatoe** [comment]
> They're the same thing, like `v-on` and `@`

If thats the case which would you prefer? its not an issue to standardize them.

**Comment by KaelWD** [comment]
We use shorthand for the others so `#` I guess

**Comment by DRoet** [comment]
> > They're the same thing, like `v-on` and `@`
> 
> If thats the case which would you prefer? its not an issue to standardize them.

I personally prefer `#`, but one might argue that it's harder to google if people have questions... so maybe `v-slot` is more explicit?

**Comment by jacekkarczmarczyk** [comment]
For personal projects i'm using # but docs are read by people at different level of knowledge, also v-slot is a new thing so i think v-slot would be better here. It will be easier for beginners and more advanced users could change it to # by themselves

**Comment by MajesticPotatoe** [comment]
How about for the examples, we do `v-slot` (since this is what majority see) and for the actual doc code we keep the `#` (there's only like 10 or so places in the code that use it and its mostly used internally)

**Comment by jacekkarczmarczyk** [comment]
Sure, i was actually thinking only about examples, docs core etc could be #

File: packages/docs/src/examples/combobox/advanced.vue [hunk 2]
```diff
@@ -25,8 +25,7 @@
     </template>
     <template
       v-if="item === Object(item)"
```
Comment (johnleider) [inline packages/docs/src/examples/combobox/advanced.vue]: this is throwing errors 
image

File: packages/docs/src/examples/combobox/comboboxMultiple.vue [hunk 3]
```diff
@@ -26,10 +26,7 @@
           multiple
           chips
         >
-          <template
-            slot="selection"
-            slot-scope="data"
-          >
+          <template slot:selection="data">
```
Comment (johnleider) [inline packages/docs/src/examples/combobox/comboboxMultiple.vue]: this is throwing errors 
image

File: packages/docs/src/examples/selection-controls/labelSlot.vue [hunk 4]
```diff
@@ -1,28 +1,44 @@
...
+      <template v-slot:label>
+        <div>
+          I agree that
+          <v-tooltip bottom>
+            <template v-slot:activator="{ on: tooltip }"></template>
+            <a
+              target="_blank"
+              href="http://vuetifyjs.com"
+              @click.stop
+              v-on="tooltip"
```
Comment (johnleider) [inline packages/docs/src/examples/selection-controls/labelSlot.vue]: this is throwing errors 
image

File: packages/docs/src/components/themes/Premium.vue [hunk 5]
```diff
@@ -48,22 +48,26 @@
...
-                    slot="activator"
-                    :href="template.demoUrl.length === 1 ? `${template.demoUrl[0]}?ref=vuetifyjs.com` : undefined"
-                    icon
-                    target="_blank"
-                    rel="noopener"
-                    aria-label="View Demo"
+                <template #activator="{ on: menu }">
+                  <v-tooltip
+                    bottom
+                    v-on="{ ...menu }"
```
Comment (jacekkarczmarczyk) [inline packages/docs/src/components/themes/Premium.vue]: `v-on="menu"`? it's in few other places
also you can do `#activator="{ on }"` and `v-on="on"`, renaming is required only if you want to merge listeners from diffrents slots

Comment (MajesticPotatoe) [reply packages/docs/src/components/themes/Premium.vue]: this was based off the example @KaelWD posted (where they are nested activators (menu -> tooltip). the syntax you provide doesn't work for this one. He also mentioned there was no need to spread when there is only 1 activator, which I had already went through and reverted them to `v-on="menu"`, though i think the `#activator="{ on }"` and `v-on="on"` syntax might be a little cleaner, so ill update them to that

Comment (MajesticPotatoe) [reply packages/docs/src/components/themes/Premium.vue]: in this particular example, it should be `v-on="{ ...menu, ...tooltip }"` anyways

**Pull request merged**